tests/fixtures/Command/create-requests.yml

```diff
@@ -0,0 +1,29 @@
+- input:
+    command: 'create-requests'
+    in-format: 'yaml'
+    out-format: 'xml'
+    request-file: './tests/fixtures/Command/create-requests.sample.yml'
+    response: './tests/fixtures/Command/create-requests.response.xml'
```

Comment:
I wouldn't divide test data in different files, if the worry here is for the fixture file to become too large then we can use [YAML anchors and references](https://blog.daemonl.com/2016/02/yaml.html).

Reply:
We should have different files for the `request-file` because the command should have the name of a  file and not the content.